src/createAnimatedComponent/createAnimatedComponent.tsx

```diff
@@ -546,6 +546,8 @@ export function createAnimatedComponent(
     // and later on, in componentDidUpdate, calculate translation for layout transition.
     getSnapshotBeforeUpdate() {
       if (
+        IS_WEB &&
+        this._component &&
         (this._component as HTMLElement).getBoundingClientRect !== undefined
```

Comment:
Does this also work?
```suggestion
        (this._component as HTMLElement)?.getBoundingClientRect !== undefined
```

Reply:
It does. Done in [116aa3e](https://github.com/software-mansion/react-native-reanimated/pull/5462/commits/116aa3e08d9691b21ec108e7580e3664515a552a).